src/main/resources/application.yaml

```diff
@@ -11,8 +11,8 @@ spring:
     name: Bulk Scanning
   datasource:
     driver-class-name: org.postgresql.Driver
-    url: jdbc:postgresql://${BULK_SCANNING_DB_HOST:localhost}:${BULK_SCANNING_DB_PORT:5432}/${BULK_SCANNING_DB_NAME:bulkscans}${BULK_SCANNING_DB_CONN_OPTIONS:}
-    username: ${BULK_SCANNING_DB_USER_NAME:bulkscanner}
+    url: jdbc:postgresql://${BULK_SCANNING_DB_HOST:localhost}:${BULK_SCANNING_DB_PORT:5435}/${BULK_SCANNING_DB_NAME:bulkscans}${BULK_SCANNING_DB_CONN_OPTIONS:}
+    username: ${BULK_SCANNING_DB_USER_NAME:bulkscans}
```

Comment:
bulkscans ? .... is this a mistake or has it been changed ?

Reply:
intentional, the username needs to match the db name in our local setup